public/javascripts/dbqueries.js

```diff
@@ -0,0 +1,34 @@
+const Database = require('better-sqlite3');
```

Comment:
This file appears to use `tab` delimiters instead of spaces like the other ones. Check your VSCode settings to make them consistently tabs or spaces. It doesn't matter which you pick, but make it consistent across all files